shell/browser/native_window_mac.mm

```diff
@@ -1477,61 +1476,44 @@ void ViewDidMoveToSuperview(NSView* self, SEL _cmd) {
   }
 
   if (@available(macOS 10.11, *)) {
```

Comment:
If we are changing to make the minimum version of the SDK 10.15, do we need this if?

Reply:
@jkleinsc minimum SDK and minimum supported version are not the same: we _support_ 10.10 and beyond but use the SDK from 10.15 :)